tvac/client.py

```diff
@@ -0,0 +1,66 @@
+import requests
+
+VARIANT_ENDPOINT = "http://rest.ensembl.org/vep/human/hgvs/{variant}" 
+
+class EnsemblError(Exception):
```

Comment:
Using a custom error here means that when we skip errors later, we only skip errors that match this exception type. It also allows for greater flexibility in testing.